3_javascript/javascript1/week1/A house price estimator.js

```diff
@@ -0,0 +1,25 @@
+// Peter
+let volumeInMetersPeter=8*10*10;
+let gardenSizeInM2Peter=100;
+let housePricePeter;
```

Comment:
💡 To make this more structured, you might want to consider using an object to store each person's house details. As we can talked about it in the class, it's a good idea to group related values together in an object
🌱 Using objects can further help streamline your price-checking logic and make the code more modular and readable.